content/Hardware Support/UNO/Update-the-connectivity-module-firmware-on-UNO-R4-WiFi.md

```diff
@@ -3,21 +3,84 @@ title: Update the connectivity module firmware on UNO R4 WiFi
 id: 9670986058780
 ---
 
-Learn how to update the wireless connectivity firmware on the ESP32-S3 module on the UNO R4 WiFi.
+Learn how to update the firmware of the ESP32-S3 connectivity module on the UNO R4 WiFi.
 
-Updating the firmware is required to use UNO R4 WiFi with Arduino Cloud.
+Updating the firmware is required to use UNO R4 WiFi with Arduino Cloud, and can resolve issues with UNO R4 WiFi not being detected by Arduino IDE and other development tools.
 
 In this article:
 
-* [Windows](#windows)
-* [macOS](#macos)
-* [Linux](#linux)
+* [Use the Firmware Updater in Arduino IDE](#ide)
+* [Use IoT Cloud to update the firmware](#iot)
+* [Use the updater script](#unor4wifi-updater)
+* [Run espflash directly](#espflash)
 
 ---
 
+<a id="ide"></a>
+
+## Use the Firmware Updater in Arduino IDE
+
+Updating the connectivity firmware is easy when using the Firmware Updater in Arduino IDE 2.2.1 or later.
+
+<a class="link-external" href="https://support.arduino.cc/hc/en-us/articles/360013896579-Use-the-Firmware-Updater-in-Arduino-IDE">Use the Firmware Updater in Arduino IDE</a>
```

Comment:
```suggestion
```
I would consider removing this link since there is no other relevant information for UNO R4 WiFi in that article (and the guide in this section is already tailored for the UNO R4 Wifi).

Reply:
Very true, that was a remnant from before I decided to include the full steps!